packages/components/tooltip/src/tooltip.tsx

```diff
@@ -364,6 +379,10 @@ export const Tooltip = forwardRef<TooltipProps, "div">(
       <>
         {trigger}
 
+        <ui.span id={tooltipContentId} __css={{ visibility: "hidden" }}>
```

Comment:
@ayanami77 
Sorry, my mistake. `visibility: hidden` affects the layout, so the correct one is `display: none`.